snuba/snapshots/bulk_load.py

```diff
@@ -1,12 +1,6 @@
 from abc import ABC, abstractmethod
```

Comment:
Might make sense to break this into `snuba.snapshots.loaders.{bulk,single}`?

Reply:
done